docs/change_log/index.md

```diff
@@ -5,6 +5,9 @@ Python-Markdown Change Log
 
 Under development: version 3.3.4 (a bug-fix release).
 
+* Use `style` attribute in tables for alignment instead of `align` for better CSS
+  inter-operation. The old behavior is available by setting `use_align_attribute=True` when
+  adding the extension.
```

Comment:
As this is a backward incompatible change, this can't be included in a point release (see the [Contributing Guide][0]). It will need to wait until the next MINOR release (version 3.4.0) and the note will go in the Backward Incompatible Changes section of a separate document (see the release notes for [version 3.3][1] for an example. The note can then be more lengthy and perhaps include an example of how to set the config option to restore the old behavior.

[0]: https://python-markdown.github.io/contributing/#versions
[1]: https://github.com/Python-Markdown/markdown/blob/master/docs/change_log/release-3.3.md

Reply:
Moved to release 3.4